ecosystem/sep-0010.md

```diff
@@ -124,6 +127,7 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
   * source account set to the **Server Account**
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
   * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give the **Client** time to sign transaction)
+  * memo: derived from the `memo` value passed by the **Client** if specified, omitted otherwise
```

Comment:
💡 Derived sounds like it is transformed or modified, however it is only copied.
```suggestion
  * memo: the `memo` value passed by the **Client** if specified, omitted otherwise
```